polygon/bridge/db.go

```diff
@@ -263,21 +288,63 @@ func (s *MdbxStore) PruneEventIDs(ctx context.Context, blockNum uint64) error {
 	kByte := make([]byte, 8)
 	binary.BigEndian.PutUint64(kByte, blockNum)
 
-	it, err := tx.RangeDescend(kv.BorEvents, nil, kByte, 0)
+	cursor, err := tx.Cursor(kv.BorEventNums)
+	if err != nil {
+		return err
+	}
+	defer cursor.Close()
+
+	k, _, err := cursor.Seek(kByte)
+	if err != nil {
+		return err
+	}
+
+	for {
+		if err := tx.Delete(kv.BorEventNums, k); err != nil {
+			return err
+		}
+
+		k, _, err = cursor.Next()
+		if err != nil {
+			return err
+		}
+		if k == nil {
+			return tx.Commit()
+		}
+	}
+}
+
+func (s *MdbxStore) DumpDB(ctx context.Context) error {
```

Comment:
we could move this as a test helper func in `bridge_test.go` not to pollute the code here and also the store interface